{"name": "Проверка получения заказа по его трекинговому номеру с несуществующим номером", "status": "passed", "description": "При получении заказа по его трекиноговому номеру несуществующим номером код ответа: 404, в теле ответа \"Заказ не найден\"", "steps": [{"name": "Генерирование случайного id", "status": "passed", "start": 1724956350142, "stop": 1724956350142}, {"name": "Получение заказа по его трекинговому номеру", "status": "passed", "parameters": [{"name": "track", "value": "820258736"}], "start": 1724956350142, "stop": 1724956350326}, {"name": "Проверка ответа получения заказа по его трекинговому номеру с несуществующим номером", "status": "passed", "steps": [{"name": "Проверка статус кода получения заказа по его трекинговому номеру с несуществующим номером", "status": "passed", "start": 1724956350326, "stop": 1724956350326}, {"name": "Проверка тела ответа полученного заказа по его трекинговому номеру с несуществующим номером", "status": "passed", "start": 1724956350326, "stop": 1724956350326}], "start": 1724956350326, "stop": 1724956350326}], "start": 1724956350142, "stop": 1724956350327, "uuid": "cefb693f-1b16-4b1b-903c-237c7cd0c61c", "historyId": "40062dc85c46aa83fefc2e19e0a1e611", "testCaseId": "40062dc85c46aa83fefc2e19e0a1e611", "fullName": "tests.test_get_order_by_track.TestGetOrderByTrack#test_get_order_by_track_with_not_existing_track", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order_by_track"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "DESKTOP-AS1GOJN"}, {"name": "thread", "value": "5404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order_by_track"}]}